### PR TITLE
add not in operator

### PR DESCRIPTION
**Pull request by wanghaofei90**
- Add a new operator **not in**
- Add 2 test cases

Please CR @tyrchen

### CODE REVIEW

**Comment by wanghaofei90** [comment]
I need to merge it for now, you can comment later.
@tyrchen

File: rule.jison [hunk 1]
```diff
@@ -79,6 +80,8 @@ e
         {$$ = $1 !== $3;}
     | term IN term
         {$$ = $3.indexOf($1) !== -1;}
+    | term NOT_IN term
+        {$$ = $3.indexOf($1) === -1;}
```
Comment (tyrchen) [inline rule.jison:84]: why can't we use not (country in ["us", "cn"])?

**Comment by tyrchen** [comment]
why can't we use not (country in ["us", "cn"]?

**Comment by wanghaofei90** [comment]
= has <>
in has not in
I think it's more consistent, and closer to sql like query. Talked with ops
this morning, they prefer to have not in.

What's the down side of having an extra operator?

**Pull request merged**